grpc-protocol/src/main/java/com/linecorp/armeria/client/grpc/protocol/UnaryGrpcClient.java

```diff
@@ -90,6 +90,12 @@ public UnaryGrpcClient(WebClient webClient) {
      * Constructs a {@link UnaryGrpcClient} for the given {@link WebClient} and {@link SerializationFormat}.
      * The specified {@link SerializationFormat} should be one of {@code UnaryGrpcSerializationFormats#PROTO},
      * {@code UnaryGrpcSerializationFormats#PROTO_WEB}, or {@code UnaryGrpcSerializationFormats#PROTO_WEB_TEXT}.
+     *
+     * <p>Prefer using a standard client building pattern, e.g.:</p>
+     * <pre>{@code
+     * UnaryGrpcClient client =
+     *     Clients.builder("gproto-web+http://127.0.0.1:8080").build(UnaryGrpcClient.class);
```

Comment:
nit:
```suggestion
     *     Clients.newClient("gproto-web+http://127.0.0.1:8080", UnaryGrpcClient.class);
```
Could you also update it in the tests where possible?

Reply:
Thanks!